Annex60/Fluid/HeatPumps/Carnot_y.mo

```diff
@@ -91,6 +86,24 @@ For a similar model that can be used as a chiller, see
 revisions="<html>
 <ul>
 <li>
+January 3, 2017, by Michael Wetter:<br/>
+Removed parameters
+<code>effInpEva</code> and <code>effInpCon</code>
+and updated documentation.
+This is for
+<a href=\"https://github.com/iea-annex60/modelica-annex60/issues/497\">
+issue 497</a>.
+</li>
+<li>
+August 8, 2016, by Michael Wetter:<br/>
```

Comment:
I am a bit confused whether this date is intentional chosen or if you forgot to edit it?